### DISCUSSION TITLE
Remove ventoy UEFI key

### DISCUSSION THREAD

**Discussion by Choum28** [ventoy/Ventoy#959] [Q&A] [answered]
Hello,

How can I remove the UEFI key enroll by ventoy on a pc, I found no information about that.

**Comment by Choum28** [comment, marked as answer]
solution :

Boot with Super UEFIinSecureBoot Disk (https://github.com/ValdikSS/Super-UEFIinSecureBoot-Disk)
Select efi tool
Select keytool.efi
Edit keys
Select The mahcine owner key list (MokList)
Go to mok Key you want to remove
Select delete.
reboot.

**Comment by Couch9388** [comment]
I've tried the steps today, but still having the same issue. This is the related issue: https://github.com/ventoy/Ventoy/issues/2348
